### PR TITLE
🔒 Igor/bump cryptiles to fix vulnerability

### PR DESCRIPTION
**Pull request by ibarsi**
Fixes: #29

### CODE REVIEW

File: src/codat-queries.js [hunk 1]
```diff
@@ -222,10 +222,6 @@ class BankAccountsQuery extends FlexiblePagedQueryWithDataConnection {
 exports.BankAccountsQuery = BankAccountsQuery
 
 class BankAccountTransactionsQuery extends FlexiblePagedQueryWithDataConnectionAndRecordId {
-  constructor (companyId, dataConnectionId, bankAccountId, queryString, pageNumber, pageSize) {
```
Comment (jasond-s) [inline src/codat-queries.js:225]: This seems like an unrelated change?

**Comment by ericpearson** [comment]
@jasond-s can you guys please fix this, it's impacting your customers
